web-services/map-reduce/src/main/java/datawave/webservice/mr/MapReduceBean.java

```diff
@@ -757,64 +747,58 @@ public StreamingOutput getResultFile(@PathParam("jobId") String jobId, @PathPara
         MapReduceInfoResponse result = response.getResults().get(0);
         String hdfs = result.getHdfs();
         String resultsDir = result.getResultsDirectory();
-        final FileSystem fs = getFS(hdfs, response);
+
         final Path resultFile = new Path(resultsDir, fileName);
 
-        FSDataInputStream fis;
-        try {
+        try (final FileSystem fs = getFS(hdfs, response)) {
             if (!fs.exists(resultFile) || !fs.isFile(resultFile)) {
                 NotFoundQueryException qe = new NotFoundQueryException(DatawaveErrorCode.FILE_NOT_FOUND,
                                 MessageFormat.format("{0} at path {1}", fileName, resultsDir));
                 response.addException(qe);
                 throw new NotFoundException(qe, response);
             }
 
-            fis = fs.open(resultFile);
+            try (FSDataInputStream fis = fs.open(resultFile)) {
+                // Make a final reference to the fis for referencing inside the inner class
+                final FSDataInputStream fiz = fis;
+                return new StreamingOutput() {
```

Comment:
This looks like it won't work to me. We're returning a StreamingOutput instance where we closed the `fis` (via the try-with-resources above.

When the caller attempts to use `streamingOutput.write(someOutput)`, that'll produce some kind of IOException due to the underlying input stream being closed.